README.md

```diff
@@ -125,10 +125,30 @@ Our recommended `.babelrc` config for this is here:
 
 [See documentation on the Styled Components site](https://www.styled-components.com/docs/tooling#babel-plugin)
 
-## Browser support
+## Support levels
```

Comment:
🙈 would be nice to get some emoji down here

Reply:
📊

Reply:
```suggestion
## :bar_chart: Support levels
```